test_conformance/subgroups/subhelpers.h

```diff
@@ -1080,7 +1080,7 @@ template <typename Ty>
 typename std::enable_if<TypeManager<Ty>::is_sb_scalar_type::value>::type
 set_value(Ty &lhs, const cl_ulong &rhs)
 {
-    lhs.data = rhs;
+    lhs.data = cl_half_from_float(static_cast<cl_float>(rhs), CL_HALF_RTE);
```

Comment:
This will only give us integer-valued float values and a lot of +Infs; I think we'd want coverage of all the special values (unless that's handled elsewhere).

Reply:
I thought about this too, but I don't think this is any worse after this change, is it?

Reply:
That's fair. It depends on your definition of worse I guess--we're losing coverage of fractional values, but getting around the blocker of subnormals. I'm OK with letting this in as long as we're tracking it; I'll verify that an issue was generated from the comments on the original PR.

Reply:
#1199 covers this